src/DataTable/filters/CheckboxFilter.jsx

```diff
@@ -27,7 +27,7 @@ function CheckboxFilter({
   return (
     <Form.Group role="group" aria-labelledby={ariaLabel.current}>
       <FormLabel id={ariaLabel.current} className="pgn__checkbox-filter-label">{Header}</FormLabel>
-      <Form.CheckboxSet name={Header}>
+      <Form.CheckboxSet name={Header} value={filterValue || []}>
```

Comment:
L16 has

```jsx
const checkedBoxes = filterValue || [];
```

Could/should this just be

```suggestion
      <Form.CheckboxSet name={Header} value={checkedBoxes}>
```

Reply:
Yes, good catch!